docs/learn/assets.md

```diff
@@ -99,7 +99,150 @@ To ensure the authenticity of the images, please generate the hash of the upload
 
 </details>
 
-When you metadata file is ready, upload it (centralised or decentralised storage).
+Such JSON can be generated manually or using tools, such as [`@lukso/lsp-utils`](https://github.com/lukso-network/lsp-utils). To generate the JSON from predefined fields:
+
+- Install the `@lukso/lsp-utils` library
+
+```bash
+npm i @lukso/lsp-utils
+```
+
+- Then import the `generateLSP4JSON` to generate the JSON
+
+```js
+import { generateLSP4JSON } from '@lukso/lsp-utils';
+```
+
+Example:
+
+```js
+const name = 'Test NFT';
+
+const description = 'This is a test NFT collection';
+
+const links = [
+  { title: 'Website', url: 'https://example.com' },
+  { title: 'Twitter', url: 'https://twitter.com/example' },
+];
+
+const attributes = [
+  { key: 'trait', value: 'rare', type: 'string' },
+  { key: 'level', value: '5', type: 5 },
+];
+
+const hashVerification = {
+  method: 'keccak256(bytes)',
+  data: '0x1234567890abcdef',
```

Comment:
Shouldn't it be a 32 bytes long hash? Suggest making it 32 bytes long for ensuring the code sample is more accurate